src/controller/authController.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
remove these, disabling eslint is not a best practice, instead fix the issue by following eslint suggestions